### PR TITLE
expose some APIs from orocos.rb to Syskit's Port/Reader/Writer API

### PR DESCRIPTION
**Pull request by doudou**
Depends on:
- [x] https://github.com/rock-core/tools-orocosrb/pull/123

### CODE REVIEW

**Comment by doudou** [comment]
Could you have a look, @elael ?

**Comment by doudou** [comment]
Since elael's unavailable this week, could you take care of this, @renan028 ?

**Comment by renan028** [comment]
Ok!

File: test/test_port.rb [hunk 1]
```diff
@@ -363,6 +363,110 @@ def configure
...
+        it "returns nil if the reader is not yet connected" do
+            refute @port_reader.connected?
+            assert_nil @port_reader.read_new
+        end
+        describe "without an explicitely-provided sample" do
+            before do
+                syskit_wait_ready(@port_reader)
+            end
+            it "reads new samples" do
+                @orocos_port.write(10)
```
Comment (renan028) [inline test/test_port.rb]: if I run:
syskit test test_port.rb
I get:
NoMethodError: undefined method `write' for stub67.out:Orocos::OutputPort

File: test/test_port.rb [hunk 2]
```diff
@@ -363,6 +363,110 @@ def configure
...
+                assert_equal 10, @port_reader.read
+            end
+            it "returns nil if there are no samples" do
+                assert_nil @port_reader.read
+            end
+            it "returns the last received sample if there are no new samples" do
+                @orocos_port.write(10)
+                @port_reader.read_new
```
Comment (renan028) [inline test/test_port.rb]: @port_reader.read

File: test/test_port.rb (hunk 1 above)
Comment (doudou) [reply test/test_port.rb]: This test requires https://github.com/rock-core/tools-orocosrb/pull/123

File: test/test_port.rb (hunk 2 above)
Comment (doudou) [reply test/test_port.rb]: :+1:

File: test/test_port.rb (hunk 1 above)
Comment (doudou) [reply test/test_port.rb]: Also, within the Ruby packages (i.e. outside of a bundle), just use `ruby`

~~~
ruby -Ilib -Itest test/test_port.rb
~~~

Comment (doudou) [reply test/test_port.rb]: Elael's just reviewed the orocos.rb PR. Could you update it and check if the tests pass again ?

Comment (renan028) [reply test/test_port.rb]: ok! everything is working now.

**Pull request merged**